4.analyze-components/scripts/nbconverted/1.visualize-reconstruction.r

```diff
@@ -30,10 +35,15 @@ target_vae_recon_cost_df <- compile_reconstruction_data(dataset, data_focus = "v
 
 target_vae_loss_gg <- plot_vae_training(target_vae_recon_cost_df)
 
-ggsave(file.path(base_dir, paste0("vae_training_reconstruction_", dataset, ".pdf")),
-       plot = target_vae_loss_gg, dpi = 500, height = 6, width = 4)
-ggsave(file.path(base_dir, paste0("vae_training_reconstruction_", dataset, ".png")),
-       plot = target_vae_loss_gg, height = 6, width = 4)
+for(extension in c('.png', '.pdf')) {
```

Comment:
Looks like you're doing the same thing with different datasets, is it worth it to have a for loop for this instead?

Reply:
yeah, I think it is worth making this a function. I will update! Thanks!